### DISCUSSION TITLE
Coupon can not be applied without credit/debit card

### DISCUSSION THREAD

**Discussion by haastrupea** [open-sauced/feedback#13] [Q&A] [answered]
Hi Everyone!
I registered to participate in the #100DaysOfOSS challenge and I got a coupon for a whole year.

I am in Nigeria, my local Debit card got declined and as a result could not use the coupon.

It would be great if a credit/debit card is not a requirement when a coupon giving 100% discount is applied.

**Comment by haastrupea** [comment]
Netflix, Spotify, and gPay currently use Seerbit, They have easy checkout integration that I am very familiar with. I am currently a technical reviewer in one of their program

**Comment by bdougie** [comment]
Thanks for opening this up. This is a limitation for many others and we would love more suggestions on developer payment solution in the countries affected. Also would love for folks to share other countries represented and have this limitation.

**Comment by adiati98** [comment]
I don't have credit card as well (it's not common for us in the Netherlands to have credit card). And Dutch debit cards are mostly not accepted.
PayPal was an alternative. But now not many want to accept it.

That's the only reason why I opened a digital bank account not long ago where I can use the card as credit and debit card.

I don't have any solution yet, as I was one of folks who had card limitation. But if I see something, I definitely will share here :)

**Comment by bdougie** [comment, marked as answer]
This will be resolved once this PR is merged. We will need assistance testing and confirming. https://github.com/open-sauced/app/pull/1771
